### PR TITLE
Add `Performance Points Committee` article

### PR DESCRIPTION
**Pull request by Nivalyx**
based on what was announced on http://osu.ppy.sh/home/news/2021-11-09-performance-points-star-rating-updates.

- [x] the proposed article have been tested against the contribution checklist

### CODE REVIEW

File: wiki/Performance_points/Performance_points_commitee/en.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+stub: true
+---
+
+# Performance Points Committee
+
+The **Performance Points Committee** is a panel consisting of well-versed members of the development community whose main goal is to ensure that changes in the [performance points](/wiki/Beatmap) calculation algorithm could be facilitated, reviewed, and made effective in a fast and steady flow.
```
Comment (bdach) [inline wiki/Performance_points/Performance_points_commitee/en.md:7]: URL here seems off. Performance points have a wiki page of their own.

File: wiki/Performance_points/Performance_points_commitee/en.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+The panel was first announced to the public through the [November 2021 Performance Points & Star Rating Updates newspost](https://osu.ppy.sh/home/news/2021-11-09-performance-points-star-rating-updates).
```
Comment (bdach) [inline wiki/Performance_points/Performance_points_commitee/en.md:9]: Should probably also mention that it was called into existence in response to the community survey from this year.

Comment (Nivalyx) [reply wiki/Performance_points/Performance_points_commitee/en.md]: added

File: wiki/Performance_points/Performance_points_commitee/en.md (hunk 1 above)
Comment (Nivalyx) [reply wiki/Performance_points/Performance_points_commitee/en.md]: o yeah i copied the wrong url for some reason, whoops

File: wiki/Performance_points/Performance_points_commitee/en.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+stub: true
+---
+
+# Performance Points Committee
+
+The **Performance Points Committee** is a panel consisting of well-versed members of the development community whose main goal is to ensure that changes in the [performance points](/wiki/Performance_points) calculation algorithm could be facilitated, reviewed, and made effective in a fast and steady flow.
```
Comment (cl8n) [inline wiki/Performance_points/Performance_points_commitee/en.md:7]:
```suggestion
The **Performance Points Committee** is a panel of well-versed players and developers who facilitate updates to the [performance points](/wiki/Performance_points) and [star rating](/wiki/Beatmapping/Star_rating) algorithms. Their goal is to help [osu! developers](/wiki/People/The_Team/Developers) push changes faster and with more community coordination.
```
just playing around with the order until it felt better; "facilitated" isn't usually by itself, "made effective" is sorta vague and implies more responsibility than they have i think
and split sentence between the "what" and the "why"
and link star rating as well

File: wiki/Performance_points/Performance_points_commitee/en.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+The panel was first announced to the public through the [November 2021 Performance Points & Star Rating Updates newspost](https://osu.ppy.sh/home/news/2021-11-09-performance-points-star-rating-updates) in response to the [community survey](https://osu.ppy.sh/home/news/2021-08-17-pp-sr-survey) that was being held earlier that year.
```
Comment (cl8n) [inline wiki/Performance_points/Performance_points_commitee/en.md:9]:
```suggestion
The panel was first announced to the public through the [November 2021 *Performance Points & Star Rating Updates* news post](https://osu.ppy.sh/home/news/2021-11-09-performance-points-star-rating-updates) in response to a [community survey](https://osu.ppy.sh/home/news/2021-08-17-pp-sr-survey) that was held earlier that year.
```

**Review by cl8n — COMMENTED** [review]
probably better as `wiki/People/Performance_Points_Commitee/en.md`

File: wiki/Performance_points/Performance_points_commitee/en.md (hunk 3 above)
Comment (Walavouchey) [reply wiki/Performance_points/Performance_points_commitee/en.md:7]: "facilitate and approve updates"

**Comment by Nivalyx** [comment]
yup, applied all + moved the article to "people" as suggested

**Comment by Nivalyx** [comment]
@cl8n feel free to merge this right away if everything is good to go (it should be now)

File: wiki/People/Performance_Points_Committee/en.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+---
+stub: true
+---
+
+# Performance Points Committee
+
+The **Performance Points Committee** is a panel of well-versed players and developers who facilitate and approves updates to the [performance points](/wiki/Performance_points) and [star rating](/wiki/Beatmapping/Star_rating) algorithms. Their goal is to help [osu! developers](/wiki/People/The_Team/Developers) push changes faster and with more community coordination.
```
Comment (Walavouchey) [inline wiki/People/Performance_Points_Committee/en.md:7]:
```suggestion
The **Performance Points Committee** is a panel of well-versed players and developers who facilitate and approve updates to the [performance points](/wiki/Performance_points) and [star rating](/wiki/Beatmapping/Star_rating) algorithms. Their goal is to help [osu! developers](/wiki/People/The_Team/Developers) push changes faster and with more community coordination.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/People/Performance_Points_Committee/en.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+---
+stub: true
```
Comment (Walavouchey) [inline wiki/People/Performance_Points_Committee/en.md:2]: and might as well

```suggestion
stub: true
tags:
  - pp committee
```

Comment (Nivalyx) [reply wiki/People/Performance_Points_Committee/en.md:2]: oh that would be a nice addition yeah

**Review by Walavouchey — APPROVED** [review]

**Review by cl8n — APPROVED** [review]

**Pull request merged**